packages/elements/src/tree-select/index.ts

```diff
@@ -202,12 +203,41 @@ export class TreeSelect extends ComboBox<TreeSelectDataItem> {
   @property({ type: Function, attribute: false })
   public override renderer = new TreeSelectRenderer(this);
 
+  private _max: string | null = null;
+  /**
+   * Set maximum number of selected items
+   * @param value max value
+   * @default -
+   */
+  @property({ type: String })
```

Comment:
Can't we use `@property({ type: number })` instead?

Reply:
Align with `number-field` element.